### ISSUE TITLE
Parameters missing from the FAQ.

### ISSUE THREAD

**Issue by pierluigilenoci** [kubernetes/autoscaler#7470]
**Which component are you using?**:

cluster-autoscaler

**What version of the component are you using?**:

Component version: All

**What k8s version are you using (`kubectl version`)?**:

Not relevant

**What environment is this in?**:

Not relevant

**What did you expect to happen?**:

To have all the Autoscaler parameters documented.

**What happened instead?**:

At least 2 parameters were missing.

**How to reproduce it (as minimally and precisely as possible)**:

Search for `node-deletion-delay-timeout` or `node-deletion-batcher-interval` in the [FAQ](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-the-parameters-to-ca) page.

**Anything else we need to know?**:

Nope

**Comment by Shubham82** [comment]
Hi @pierluigilenoci, there is already a PR #7014 opened for adding the all parameters in FAQ.md.
PTAL!

**Comment by pierluigilenoci** [comment]
@Shubham82 Well, let's merge it then. 🚀

**Comment by Shubham82** [comment]
> @Shubham82 Well, let's merge it then. 🚀

Yeah, waiting for other members' opinions on Log flags, I hope it will mere soon.

**Comment by pierluigilenoci** [comment]
@Shubham82 any feedback?

**Comment by Shubham82** [comment]
Hi @pierluigilenoci,
JFI: PR #7014 is merged now
